### DISCUSSION TITLE
Disable maxlag for interactive tools

### DISCUSSION THREAD

**Discussion by dpriskorn** [LeMyst/WikibaseIntegrator#122] [Q&A] [answered]
Hi, see https://www.wikidata.org/wiki/Wikidata:Contact_the_development_team#QS_restrictions_removed?

I would like to disable maxlag for WBI in my tool LexUse. 
Is that possible with WBI?
I see no documentation for how to do that. 
I can send a pull request if you point me in the right direction. 😃

**Comment by dpriskorn** [comment, marked as answer]
I looked in config and found  config['MAXLAG'].
So setting config['MAXLAG'] = 0 should work I guess.

**Reply by LeMyst** [reply]
Hello,

The MAXLAG option is only used for the "wbeditentity"/ItemEngine.write(), there is not default maxlag.

**Reply by dpriskorn** [reply]
I don't understand. is it possible to adjuust maxlag or not?

**Reply by LeMyst** [reply]
I've created PR #124 to allow disabling maxlag parameter if maxlag is set to 0.
I've added maxlag to more functions.
